### DISCUSSION TITLE
Ferramentas de Ux Design no Linux

### DISCUSSION THREAD

**Discussion by devigor** [frontendbr/forum#554] [Perguntas e Respostas]
Estudando Ux Design me deparei com diversas ferramentas fodas demais para a criação de protótipos navegáveis, como: Axure, Sketch, etc. Porém essas ferramentas só possuem versões para Windows ou Mac.
Gostaria de saber se vocês que utilizam Linux e trabalham com Ux Design, o que recomendam para substituir essas ferramentas?

**Comment by calaca** [comment]
Ainda não vi uma ferramenta de prototipação feita pro Linux que fosse boa. A melhor que eu tenho conhecimento é a [UXPin](https://www.uxpin.com/) que é online.

**Comment by cristofersousa** [comment]
Ferramentas online como:

- [Cacoo](https://cacoo.com/)
- [Figma](https://www.figma.com/) 
- [Concept](http://concept.ly/)

**Comment by cristofersousa** [comment]
Inclusive olhando o que temos de gratuito por aí nessa parte é complicado encontrar algo de qualidade conforme a @calaca  disse pense em adquirir uma conta do UXPin agora se quer arriscar algo online e free eu uso o Cacco para wireframes de baixa fidelidade. 

Outros links que possam dar uma luz:

1. https://uxapprentice.wordpress.com/2012/01/04/10-free-wireframing-tools-for-designers/
2. https://www.sitepoint.com/linux-design-tools/ 
3. https://www.webdesignerdepot.com/2013/08/30-essential-ux-tools/

**Comment by casssiahaluu** [comment]
Cara. Não é bem voltado para UX, mas duas ferramentas que me ajudam muito
são InkScape (na minha humilde opinião é mais simples e melhor em qualidade
do que CorelDraw) e o GIMP. O último tem inúmeros packs e libraries pela
internet afora para deixar mais semelhante à essas outras ferramentas, me
adaptei muito mais facilmente a ele do que ao Photoshop nas versões de dois
anos atrás, por exemplo. Atualmente nem uso mais Windows pra trabalhar.

**Comment by delete** [comment]
Tem o [draw.io](http://draw.io) para wireframes também.

**Comment by filipelinhares** [comment]
Já passei pelo mesmo drama e fiquei muito tempo fuçando na internet atrás de uma ferramenta legal.
No fim estou utilizando o [Figma](figma.com), ele é muito completo e simples de usar.

Também testei:
- http://atomic.io
- http://quirktools.com
- https://vectr.com
> extra: https://hackdesign.org/toolkit

**Comment by franchagas** [comment]
Tem uma boa ferramenta que eu utilizo e funciona muito bem. 
Pencil Project 
https://pencil.evolus.vn/

**Comment by Ilzomariano** [comment]
https://wireframesketcher.com/

http://www.foreui.com/

**Comment by jjpaulo2** [comment]
Queria ressuscitar o tópico pra dizer que foi iniciado o desenvolvimento de uma ferramenta de design UX/UI nativa para Linux, chamada [Akira](https://github.com/akiraux/Akira). Ainda está em fase inicial, então o projeto tá precisando de desenvolvedores. Queria muito poder contribuir, mas ainda estou dando meus primeiros passos no mundo das contribuições open source, então fica ai o chamado pra quem se garantir e tiver interesse.
